src/licensedcode/data/licenses/saxix-mit.LICENSE

```diff
@@ -0,0 +1,32 @@
+---
+key: saxix-mit
+short_name: Stefano Apostolico MIT License
+name: Stefano Apostolico MIT License
+category: Permissive
+owner: Stefano Apostolico
+homepage_url: https://github.com/saxix/django-adminactions/blob/develop/LICENSE
+spdx_license_key: LicenseRef-scancode-saxix-mit
```

Comment:
To avoid false positive with real MIT licenses, I would put a minimum_coverage and some notes to explain the license:
```suggestion
spdx_license_key: LicenseRef-scancode-saxix-mit
minimum_coverage: 95
notes: this is the MIT license text with an extra clause wrt. commercial usage notification.
```

BTW, I strongly challenge the fact this is permissive. This license would be a major burden for any downstream user, so we need to tag this as proprietary or source available.

Reply:
"major burden" ?  I disagree.  "Any use in a commercial product must be notified to the author by email
indicating company name and product name."  It's just a required notification, and does not indicate that any permission is required. It's more of a special obligation than a restriction.  I prefer to leave it as-is.

Reply:
I am fine with the suggested notes.